QUEST_LV_0100.tsv

```diff
@@ -903,7 +903,7 @@ QUEST_LV_0100_20150317_000898	Alright, bring the Seal of Space from the Sventove
 QUEST_LV_0100_20150317_000899	You must know that the revelation is hidden in the sanctuary. {nl} The Seal of Space is the only key that will get you to the sanctuary. {nl}
 QUEST_LV_0100_20150317_000900	It's not just Tenet Chapel. {nl} You can enter all the places that the Goddess hid.
 QUEST_LV_0100_20150317_000901	The Seal of Space can only be used by the Revelator, so you won't be able to find the revelation right away. {nl}
-QUEST_LV_0100_20150317_000902	But the enemy is the Demon Queen. Even the Paladin Master couldn't defeat her.{nl}I should use the power of the Broken Altar.
+QUEST_LV_0100_20150317_000902	But the enemy is a Demon Lord. Even the Paladin Master couldn't defeat her.{nl}I should use the power of the Broken Altar.
```

Comment:
The previous translation is correct. There is only one Demon Queen, it's not a Demon Lord.